dependencies.gradle

```diff
@@ -50,7 +49,8 @@ ext.deps = [
         fastscroll: 'me.zhanghai.android.fastscroll:library:1.1.4',
         jsch: 'com.jcraft:jsch:0.1.55',
         jgit: 'org.eclipse.jgit:org.eclipse.jgit:3.7.1.201504261725-r',
-        leakcanary: 'com.squareup.leakcanary:leakcanary-android:2.3',
```

Comment:
This looks like a copy-paste problem, the `plumber` dependency is duplicated.

Reply:
Yeah it is. I saw plumber going to work in my log so didn't think to check twice. Fixed now.